tests/integration/test_ols.py

```diff
@@ -46,6 +46,22 @@ def test_feedback() -> None:
     assert response.json() == {"status": "feedback received"}
 
 
+def test_feedback_wrong_request() -> None:
```

Comment:
Feedback has its own endpoint. Shouldn't this be in a separate test file?

Reply:
if yes, then I'll move the `test_feedback` there to. Is it ok/expected?